src/keystore.c

```diff
@@ -44,16 +44,16 @@ static bool _is_unlocked_bip39 = false;
 static uint8_t _retained_bip39_seed[64] = {0};
 
 #ifdef TESTING
-void mock_state(const uint8_t* retained_seed, const uint8_t* retained_bip39_seed)
+void mock_state(const uint8_t* seed, size_t seed_len, const uint8_t* bip39_seed)
```

Comment:
how about renaming this to a `mock_keystore_state`, or better yet a `mock_keystore_unlocked_state`. last time i was browsing tests, it was quite disorienting due to naming.

Reply:
Done in a separate commit